src/rc.js

```diff
@@ -1,4 +1,5 @@
 // Copyright (C) 2017 - Texas Instruments, Jason Kridner
+// exports not changed to module.exports
```

Comment:
Why?

Reply:
I saw that file was not used , the line  'var rc = require('./rc');' was commented in src/index.js
I will replace the exports in src/rc.js too when making the other changes.